cime/config/e3sm/allactive/config_pesall.xml

```diff
@@ -8446,4 +8446,78 @@
       </pes>
     </mach>
   </grid>
+  <grid name="a%T62.+oi%oARRM60to10|a%TL319.+oi%oARRM60to10">
```

Comment:
I would suggest to add these changes for a specific machine. Also, @apcraig: we should talk about the improvements you experienced with these settings (we can do this by email).

Reply:
Good idea.  I will take care of that.